inc/namespace.php

```diff
@@ -9,189 +9,14 @@
 use WP_User;
 use WP_Http;
```

Comment:
May want to review these to check we're using them all now that you've split the functions; also alphabetise if you change

Reply:
I wonder if there's a sniff we could do for that too?